derived/derivation.adoc

```diff
@@ -134,6 +139,9 @@ We need to omit the parameter if we don't want to limit the depth.
 ==== Result
 {chunk} containing all derived nodes according to `nodeIds`, `languages`, and `depthLimit` parameters.
 
+[horizontal,labelwidth=12]
+QUESTION:: The `depth:Limit` describes the depth of the retrieved nodes (derived nodes may be full trees), the next line suggests that `depthLimit` applies to the node id's sent. Needs clarification.
```

Comment:
The idea is:
1. We select the same original nodes as with Bulk API retrieve
2. For all these nodes, we find their derivations (if available)
3. For all derivations, we return the derivations and all descendants thereof with infinite depth.
   The assumption is that derived nodes only contain other nodes that are part of the derived information.